hoomd/mpcd/StreamingGeometry.cc

```diff
@@ -37,6 +37,18 @@ void export_PlanarPoreGeometry(pybind11::module& m)
         .def_property_readonly("no_slip", &PlanarPoreGeometry::getNoSlip);
     }
 
+void export_RotatedParallelPlateGeometry(pybind11::module& m)
+    {
+    pybind11::class_<RotatedParallelPlateGeometry, std::shared_ptr<RotatedParallelPlateGeometry>>(
+        m,
+        RotatedParallelPlateGeometry::getName().c_str())
+        .def(pybind11::init<Scalar, Scalar, Scalar, bool>())
+        .def_property_readonly("H", &RotatedParallelPlateGeometry::getH)
```

Comment:
I recently renamed this from `H` to `separation`:

https://hoomd-blue--1763.org.readthedocs.build/en/1763/module-mpcd-geometry.html#hoomd.mpcd.geometry.ParallelPlates

so I suggest we call it the same here. We want to leave this as `m_H` internally for efficiency, though, so I would do the following:
1. Rename `H` to `separation` in the C++ constructor. Assign `m_H(Scalar(0.5) * separation)` in the initialization list.
2. Rename `getH` to `getSeparation` and have it return `Scalar(2)*m_H`.
3. Change this python export to reflect that:
```suggestion
        .def_property_readonly("separation", &RotatedParallelPlateGeometry::getSeparation)
```